05/DB.test.js

```diff
@@ -1 +1,78 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('insert', () => {
+
+
+    // Nie działa zgodnie z zamierzeniem
```

Comment:
Zwróć uwagę, że tutaj mamy obietnice, a nie rzucanie błędów. Inaczej to obsługujemy :)